src/tests/ftest/io/ior_small.yaml

```diff
@@ -10,7 +10,7 @@ hosts:
             - boro-F
             - boro-G
             - boro-H
-timeout: 1200
+timeout: 3000
```

Comment:
This is a huge increase.  We should understand why the test is timing out in this PR only before we land such an increase.